trajectory_msgs/JointTrajectory trajectory
---
bool done